published-namespaces.txt

```diff
@@ -598,6 +602,15 @@ Xamarin.Google.ErrorProne.Annotations
 Xamarin.Google.ErrorProne.Annotations.Concurrent
 Xamarin.Google.FlatBuffers
 Xamarin.Google.FlatBuffers.Reflection
+Xamarin.Google.Inject
+Xamarin.Google.Inject.Binder
+Xamarin.Google.Inject.Internal.AOP
+Xamarin.Google.Inject.Internal.Util
+Xamarin.Google.Inject.Matcher
+Xamarin.Google.Inject.Multibindings
```

Comment:
Reviewing note: Google has chosen to capitalize this as `Multibindings`, so this is correct because we want the capitalization of the namespace to match the class name(s).

https://google.github.io/guice/api-docs/4.1/javadoc/com/google/inject/multibindings/MultibindingsScanner.html